api/backend/blueflood/blueflood.go

```diff
@@ -119,8 +130,11 @@ func (b *blueflood) FetchSingleSeries(request api.FetchSeriesRequest) (api.Times
 	if !ok {
 		return api.Timeseries{}, fmt.Errorf("unsupported SampleMethod %s", request.SampleMethod.String())
 	}
+	queryResolution := b.config.bluefloodResolution(
+		time.Duration(request.Timerange.Resolution()),
```

Comment:
The case the `time.Duration` should be unnecessary, since `Timerange.Resolution()` now returns a `time.Duration` object.